### PR TITLE
Reduce default log level to 2

### PR DESCRIPTION
**Pull request by wongma7**
**Is this a bug fix or adding new feature?** fix https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/252

**What is this PR about? / Why do we need it?** 5 is meant for debugging, https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md 

I ran make generate-kustomize.

**What testing is done?**

### CODE REVIEW

**Comment by ayberk** [comment]
/lgtm

We need this for other CSI drivers as well.

File: deploy/kubernetes/base/controller-serviceaccount.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: efs-csi-controller-sa
-    namespace: sample
+    namespace: default
```
Comment (ayberk) [inline deploy/kubernetes/base/controller-serviceaccount.yaml:50]: Do we want this?

File: deploy/kubernetes/base/controller-deployment.yaml [hunk 2]
```diff
@@ -37,9 +37,8 @@ spec:
           args:
             - --endpoint=$(CSI_ENDPOINT)
             - --logtostderr
-            - --v=5
-            # Uncomment below line to allow access point root directory to be deleted by controller.
-            #- --delete-access-point-root-dir
+            - --v=2
+            - --delete-access-point-root-dir=false
```
Comment (ayberk) [inline deploy/kubernetes/base/controller-deployment.yaml:41]: Do we want this?

File: deploy/kubernetes/base/controller-serviceaccount.yaml (hunk 1 above)
Comment (wongma7) [reply deploy/kubernetes/base/controller-serviceaccount.yaml:50]: IDK where it came from. it's make generate-kustomize at work.

I think default makes more sense than sample though?

File: deploy/kubernetes/base/controller-deployment.yaml (hunk 2 above)
Comment (wongma7) [reply deploy/kubernetes/base/controller-deployment.yaml:41]: make generate-kustomize cleans out comments. It's hard to preserve them. TBH, I think it's better if we try to copy what @AndyXiangLi  did for ebs and document these flags in README than keep comments that won't be noticed https://github.com/kubernetes-sigs/aws-ebs-csi-driver#driver-options

Comment (wongma7) [reply deploy/kubernetes/base/controller-deployment.yaml:41]: I misread, thought you meant just the comment being removed.

about it being false by default:

yes, that's intended, it comes from the helm chart making it false by default.

Comment (wongma7) [reply deploy/kubernetes/base/controller-deployment.yaml:41]: one more comment:

about it being false by default in the helm chart in the first place:

discussion is here: https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/411

File: deploy/kubernetes/base/controller-serviceaccount.yaml (hunk 1 above)
Comment (ayberk) [reply deploy/kubernetes/base/controller-serviceaccount.yaml:50]: Yeah agreed. Just wanted to make sure we don't include something we don't want.

**Pull request merged**